source/ios/AdaptiveCards/AdaptiveCards/AdaptiveCards/ACRView.mm

```diff
@@ -370,8 +370,13 @@ - (void)loadImage:(std::string const &)urlStr
     }
     
     NSString *nSUrlStr = [NSString stringWithCString:urlStr.c_str()
-                                          encoding:[NSString defaultCStringEncoding]];
+                                            encoding:[NSString defaultCStringEncoding]];
     NSURL *url = [NSURL URLWithString:nSUrlStr];
+    // if url is nil, try again with adding base url from host config
```

Comment:
>if url is nil, try again with adding base url from host config [](start = 7, length = 62)

This comment is wrong: you're not checking whether the URL is nil, but whether it's relative.

Reply:
thanks